src/controller/model/device.ts

```diff
@@ -712,6 +714,10 @@ class Device extends Entity {
         }
     }
 
+    public async getNetworkParameters(): Promise<NetworkParameters> {
+        return Entity.adapter.getNetworkParameters();
```

Comment:
This only seems to be used in a test case: https://github.com/Koenkk/zigbee-herdsman/pull/518/files#diff-143091d444c5f0ff7479e674fcbc691161665ccf435ad8c8c7d8f55d20b8361aR3924

Would be nice if this method can be removed

Reply:
@Koenkk Hmm slight blunder from my side - I use the getNetworkParameters() from the converter: https://github.com/Koenkk/zigbee-herdsman-converters/blob/09495aca82b6498d2222664b7edff5cf78b70a7d/devices/schneider_electric.js#L57

Unless there is a better way of getting the network channel  from a converter I think we should re-add the method.